packages/babel-plugin/src/index.tsx

```diff
@@ -11,16 +12,58 @@ export default declare<State>((api) => {
   return {
     inherits: require('babel-plugin-syntax-jsx'),
     visitor: {
+      Program(path, state) {
+        const importDeclarations = path.node.body.filter((node) => t.isImportDeclaration(node));
+        if (
+          importDeclarations.some(
+            (declaration: any) => declaration.source.value === '@compiled/core'
```

Comment:
these `any`s are not ideal, but if I use what I believe is the correct type `t.ImportDeclaration` I get a TS error that I'm finding it a little hard to unpack 😓 

```
Argument of type '(declaration: t.ImportDeclaration) => boolean' is not assignable to parameter of type '(value: Statement, index: number, array: Statement[]) => unknown'.
  Types of parameters 'declaration' and 'value' are incompatible.
    Type 'Statement' is not assignable to type 'ImportDeclaration'.
      Type 'BlockStatement' is missing the following properties from type 'ImportDeclaration': specifiers, source, importKind
```